src/vs/platform/terminal/node/ptyHostService.ts

```diff
@@ -187,13 +187,13 @@ export class PtyHostService extends Disposable implements IPtyService {
 		const logChannel = client.getChannel(TerminalIpcChannels.Log);
 		LogLevelChannelClient.setLevel(logChannel, this._logService.getLevel());
 		const ptyHostLogResource = URI.file(join(this._environmentService.logsPath, `${TerminalLogConstants.FileName}.log`));
-		this._loggerService.registerLoggerResource({ id: 'ptyHostLog', name: localize('ptyHost', "Pty Host"), resource: ptyHostLogResource });
+		this._loggerService.registerLogger({ id: 'ptyHostLog', name: localize('ptyHost', "Pty Host"), resource: ptyHostLogResource });
```

Comment:
@Tyriar In your case you have to register the logger with the main logger service because your logger is created in a different process. Registering shall be sufficient for the system to create output channel for your logger.